lms/djangoapps/mobile_api/social_facebook/groups/views.py

```diff
@@ -0,0 +1,123 @@
+_FACEBOOK_API_VERSION = settings.FACEBOOK_API_VERSION
+_FACEBOOK_APP_ID = settings.FACEBOOK_APP_ID
+_FACEBOOK_APP_SECRET = settings.FACEBOOK_APP_SECRET
+
+@mobile_view()
+class Groups(generics.CreateAPIView, mixins.DestroyModelMixin):
+    """
+    **Use Case**
+
+        An API to Create or Delete course groups.
+
+    **Creation Example request**:
+
+        POST /api/mobile/v0.5/social/facebook/groups/<group_id>
+
+        Parameters: name : string,
+                    description : string, 
+                    privacy : open/closed
+
+    **Creation Response Values**
+
+        {"group-id": group_id}
+
+
+    **Deletion Example request**:
+
+        DELETE /api/mobile/v0.5/social/facebook/groups/<group_id>
+
+    **Deletion Response Values**
+
+        {"success" : "true"}
+
+    """
+    serializer_class = serializers.GroupSerializer
+
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.DATA, files=request.FILES)
+        if serializer.is_valid():
+            post_args = request.POST.dict()
+            try:
+                graph = facebook.GraphAPI(facebook.get_app_access_token(_FACEBOOK_APP_ID, _FACEBOOK_APP_SECRET))
+                app_groups_response = graph.request(_FACEBOOK_API_VERSION + '/' + _FACEBOOK_APP_ID + "/groups", post_args=post_args)
+            except facebook.GraphAPIError, ex:
+                return Response({'error': ex.result['error']['message']}, status=status.HTTP_400_BAD_REQUEST)
+
+            return Response(app_groups_response)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    def delete(self, request, *args, **kwargs):
+        try:
+            graph = facebook.GraphAPI(facebook.get_app_access_token(_FACEBOOK_APP_ID, _FACEBOOK_APP_SECRET))
+            post_args = {'method': 'delete'}
+            return Response(graph.request(_FACEBOOK_API_VERSION + '/' + _FACEBOOK_APP_ID + "/groups/" + kwargs['group_id'], post_args=post_args))
+        except facebook.GraphAPIError, ex:
+            return Response({'error': ex.result['error']['message']}, status=status.HTTP_400_BAD_REQUEST)
+
+
+@mobile_view()
+class GroupsMembers(generics.CreateAPIView, mixins.DestroyModelMixin):
+    """
+    **Use Case**
+
+        An API to Invite and Remove members to a group
+
+    **Invite Example request**:
+
+        POST /api/mobile/v0.5/social/facebook/groups/<group_id>/member/<member_id>
+
+        Parameters: members : int,int,int... 
+
+
+    **Invite Response Values**
+
+        {"member_id" : success/error_message}       A response with each member_id and whether or not
+                                                    the member was added successfully. If the member was 
+                                                    not added successfully the Facebook error message is provided. 
+        
+    **Remove Example request**:
+
+        DELETE /api/mobile/v0.5/social/facebook/groups/<group_id>/member/<member_id>
```

Comment:
Hmm.. Can you please remove any endpoints that are not currently used by the app or put a comment to that effect?  This will help us with future maintenance of the code.  Thanks.